cli.js

```diff
@@ -48,133 +43,60 @@ var cli = meow([
 	]
 });
 
-var rejectionCount = 0;
-var exceptionCount = 0;
-var testCount = 0;
-var fileCount = 0;
-var errors = [];
-
-function prefixTitle(file) {
-	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
-
-	var base = path.dirname(cli.input[0]);
-
-	if (base === '.') {
-		base = cli.input[0] || 'test';
-	}
-
-	base += path.sep;
-
-	var prefix = path.relative('.', file)
-		.replace(base, '')
-		.replace(/\.spec/, '')
-		.replace(/test\-/g, '')
-		.replace(/\.js$/, '')
-		.split(path.sep)
-		.join(separator);
-
-	if (prefix.length > 0) {
-		prefix += separator;
-	}
-
-	return prefix;
-}
+updateNotifier({pkg: cli.pkg}).notify();
 
-function stats(stats) {
-	testCount += stats.testCount;
+if (cli.flags.init) {
+	require('ava-init')();
+	process.exit();
```

Comment:
You can't `process.exit()` here as `ava-init` won't have a time to finish. Just `return` and XO will pass when https://github.com/sindresorhus/ava/pull/277/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R132 lands.